nestkernel/nest_time.h

```diff
@@ -492,11 +492,11 @@ class Time
   double
   get_ms() const
   {
-    if ( tics == LIM_POS_INF.tics )
+    if ( tics >= LIM_POS_INF.tics )
```

Comment:
I agree with @heplesser.

In my opinion this calls for a macro and/or a function (e.g. `IS_POSINFINITY()` `preserve_infinity()`). But this is beyond the scope of this PR and would require additional changes to the time class to handle the different infinity value checks in a more generic way.